src/__tests__/articleTests/viewArticle.test.js

```diff
@@ -54,3 +55,14 @@ it("viewing article", () => {
   let component = shallow(<ViewArticle {...props} />);
   component;
 });
+
+it("email share component", () => {
+  let dispatch = jest.fn();
+  let handleEmail = jest.fn();
+  let props = {
+    dispatch,
+    handleEmail
+  };
+  let component = shallow(<EmailModal {...props} />);
+  component;
+});
```

Comment:
What does calling component do?

Reply:
I was using component a variable for holding the particular component that I render for testing.

Reply:
Do you `expect` anything to happen. When you're writing tests, you often need to check that values meet certain conditions using expect